agent/util-scripts/pbench-move-results

```diff
@@ -186,10 +186,14 @@ if [[ ! -z "${show_server}" ]] ;then
 fi
 
 # ssh probe test
-ssh -n -q -i ${pbench_bin}/id_rsa ${ssh_opts} ${results_repo} exit
-if [[ $? -ne 0 ]]; then
-    error_log "ERROR: results host unreachable: ${results_repo}"
-    debug_log "the following ssh command failed: \"ssh -n -q -i ${pbench_bin}/id_rsa ${ssh_opts} ${results_repo} exit\""
+ssh -n -i ${pbench_bin}/id_rsa ${ssh_opts} ${results_repo} exit
+sts=${?}
+if [[ ${sts} -eq 255 ]]; then
+    error_log "ERROR: ssh command, \"ssh -n -i ${pbench_bin}/id_rsa ${ssh_opts} ${results_repo} exit\", failed"
```

Comment:
It would be better if you captured the command in a variable which you used to execute the command and then also used to report errors with it.

Reply:
Yes, it would be, but then we'd have to be sure we properly handle all the resolved variables.  Doesn't seem worth all that effort for this legacy code.